styles/style.css

```diff
@@ -0,0 +1 @@
+body { font-family: 'Gotham Pro', sans-serif;}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/r4/pullRequest?prid=5887703)